### DISCUSSION TITLE
All nodes dropped using multiple start and end location with time windows

### DISCUSSION THREAD

**Discussion by benbaccarw** [google/or-tools#4134] [Routing (and legacy CP) questions]
Hello,

I am working on a routing problem.

In my routing problem I want some of the nodes to not be included in the same route.
To do that I have set multiple vehicle with different time windows, each vehicle start and end node Is available during a certain time. 

Indeed, each vehicle represents a day so for example, if I have 2 vehicles:
* The first one will have a start node available from `[28800,28800]` and end node available from `[80000,80000]` this represents `8 am` and `10 pm` for the day one in seconds.
* The second vehicle will have a start node available from `[115200,115200]` and end node available from `[166400,166400]` this represents `8 am` and `10 pm` for the day two.

The visited node can be visited either from `28800` to `80000` or `115200` to `166400` depending on whether they can be visited together or not.

Also, I have set penalty for node so they can be dropped.

The things is that without the time windows using the path cheapest arc heuristics method give me a solution pretty quick, but when I am adding time windows especially on the start and end node it drops all the node even after running for 5 hours.

Could you help me solve that please ? Don't hesitate to ask more information if it is not clear.

Thank you !

**Comment by jmarca** [comment]
I’m pretty sure the vehicle times are impossible. Make a gap of at least one between start and end. I can’t find the API offhand, but I thought it was >= lower bound and < upper bound.
